tests/sparse_test.py

```diff
@@ -2548,17 +2548,18 @@ def test_matmul(self, shape, dtype, Obj, bshape):
       self.assertAllClose(M @ x, Msp @ x, rtol=MATMUL_TOL)
 
   @jtu.sample_product(
+    cls=[sparse.BCOO, sparse.BCSR],
     input_type=[scipy.sparse.coo_matrix, scipy.sparse.csr_matrix,
                 scipy.sparse.csc_matrix],
     shape=[(5, 8), (8, 5), (5, 5), (8, 8)],
     dtype=jtu.dtypes.floating + jtu.dtypes.complex,
   )
-  def test_bcoo_from_scipy_sparse(self, input_type, shape, dtype):
+  def test_bcoo_bcsr_from_scipy_sparse(self, cls, input_type, shape, dtype):
     rng = rand_sparse(self.rng())
     M = rng(shape, dtype)
     M_sparse = input_type(M)
-    M_bcoo = sparse.BCOO.from_scipy_sparse(M_sparse)
-    self.assertArraysEqual(M, M_bcoo.todense())
+    M_batched = cls.from_scipy_sparse(M_sparse)
```

Comment:
minor nitpick, but I'd change `M_sparse` to `M_scipy` and change `M_batched` to `M_jax`

Reply:
Like the names!